src/test_l3_hdmi_cec_driver.c

```diff
@@ -1,409 +1,809 @@
-/**
-*  If not stated otherwise in this file or this component's LICENSE
-*  file the following copyright and licenses apply:
-*
-*  Copyright 2023 RDK Management
+/*
+* If not stated otherwise in this file or this component's LICENSE file the
+* following copyright and licenses apply:*
+* Copyright 2024 RDK Management
 *
-*  Licensed under the Apache License, Version 2.0 (the License);
-*  you may not use this file except in compliance with the License.
-*  You may obtain a copy of the License at
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
 *
-*  http://www.apache.org/licenses/LICENSE-2.0
+* http://www.apache.org/licenses/LICENSE-2.0
 *
-*  Unless required by applicable law or agreed to in writing, software
-*  distributed under the License is distributed on an AS IS BASIS,
-*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-*  See the License for the specific language governing permissions and
-*  limitations under the License.
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
 */
+/**
+ * @addtogroup HPK Hardware Porting Kit
+ * @{
+ *
+ */
+/**
+ * @addtogroup HDMI_CEC HDMI CEC Module
+ * @{
+ *
+ */
+/**
+ * @defgroup HDMI_CEC_HALTESTS HDMI CEC HAL Tests
+ * @{
+ *
+ */
 
 /**
-* @file TODO: test_l3_hdmi_cec_driver.c
-* @page module_name TODO: Required field, name of the main module
-* @subpage sub_page_name TODO: Add a function group if relevant
-*
-* ## Module's Role
-* TODO: Explain the module's role in the system in general
-* This is to ensure that the API meets the operational requirements of the module across all vendors.
-*
-* **Pre-Conditions:**  TODO: Add pre-conditions if any@n
-* **Dependencies:** TODO: Add dependencies if any@n
-*
-* Ref to API Definition specification documentation :[hdmi-cec_halSpec.md](../../docs/pages/hdmi-cec_halSpec.md)
-*/
+ * @defgroup HDMI_CEC_HALTESTS_L3 HDMI CEC HAL Tests L2 File
+ * @{
+ * @parblock
+ *
+ * ### L2 Test Cases for HDMI CEC HAL :
```

Comment:
remove L2 and L1 relegated the info in this file

Reply:
done